app/gui/src/main/scala/org/bitcoins/gui/AppMenuBar.scala

```diff
@@ -11,14 +12,30 @@ import scalafx.stage.FileChooser.ExtensionFilter
 import java.io.File
 import scala.util.Properties
 
+// Utility functions for menuing. May want in a util file somewhere
+object menuUtil {
```

Comment:
nit: Upper case `MenuUtil`

Reply:
I removed menuUtil in favor of using the builtin KeyCombination.ShortcutDown which covers the use case